### PR TITLE
Implementing the Debug trait for Server, required when one wants to c…

### PR DESCRIPTION
**Pull request by hammett**
…alls unwrap_err on a Result<Server, Error>

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

PTAL @BusyJay

File: src/server.rs [hunk 1]
```diff
@@ -341,6 +343,12 @@ impl Inner {
     }
 }
 
+impl Debug for Inner {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        write!(f, "Server {:?}", self.bind_addrs)
```
Comment (overvenus) [inline src/server.rs]: I think `Formatter::debug_struct`[0] is better than handwritten.

[0] https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct

Comment (BusyJay) [reply src/server.rs]: I think it's fine, since only one field is used.

**Comment by siddontang** [comment]
Thanks @hammett

**Pull request merged**